### PR TITLE
Update dao-entrance-phase-1.md

### PR DESCRIPTION
**Pull request by bai-3**
I realized that because the md we submitted contains the img tag, there was a reat failure message in the static rendering(The img tag contains style="800px"). I submitted this repair request. I'm very sorry

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]
Thanks for the update.

**Review by dsm-w3f — APPROVED** [review]

**Comment by bai-3** [comment]
I find it very difficult to merge this pr. Can you fix it directly and then close the pr @Noc2 @0xCaso @dsm-w3f. Thank you very much

**Comment by Noc2** [comment]
> I find it very difficult to merge this pr. Can you fix it directly and then close the pr @Noc2 @0xCaso @dsm-w3f. Thank you very much

We will merge it as soon as it has three approvals from committee members, just to follow the official process. You don't need to do anything else. Thanks a lot.

**Review by takahser — APPROVED** [review]

**Pull request merged**
